### PR TITLE
creating usertoken

### PR DESCRIPTION
**Pull request by maniplek**
What does this PR do?
This PR aims to creating user token 

Description of Task to be completed?
after creating a user we must get a token 

How should this be manually tested?
Pull this project repository
Create an environment variable with python version 3.10
Run pip install -r requirements.txt

### CODE REVIEW

**Review by SamuelIZABAYO — CHANGES_REQUESTED** [review]
A file should have one empty line at the end

File: apps/users_app/admin.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+from .models import CustomUser
+from django.apps import apps
+# Register your models here.
+
+admin.site.register(CustomUser)
+
+app = apps.get_app_config('graphql_auth')
+
+for model_name, model in app.models.items():
+    admin.site.register(model)
```
Comment (kagabof) [inline apps/users_app/admin.py:11]: Please add an empty line at the end of this file.

File: apps/users_app/schema/query/query.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    user = graphene.Field(UserType, id=graphene.Int())
+    
+    def resolve_user(self, info, **kwargs):
+        id = kwargs.get('id')
+
+        if id is not None:
+            return CustomUser.objects.get(pk=id) 
+
+        return None
+
```
Comment (kagabof) [inline apps/users_app/schema/query/query.py:16]: Please add an empty line at the end of this file.

File: scrum/settings.py [hunk 3]
```diff
@@ -136,7 +141,35 @@
...
+
+GRAPHQL_AUTH = {
+    'LOGIN_ALLOWED_FIELDS': ['email', 'username'],
+    # ...
+}
+
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
 with open('.env') as f:
     SECRET_KEY = f.read().strip()
```
Comment (kagabof) [inline scrum/settings.py:175]: Please add an empty line at the end of this file.

**Review by kagabof — CHANGES_REQUESTED** [review]
well done @maniplek, Please check the feedback given and I would like you to rename the app by authentication.

**Comment by maniplek** [comment]
okey

**Pull request closed without merging**